ui/pages/send/send.test.js

```diff
@@ -163,6 +177,25 @@ describe('Send Page', () => {
       const { queryByText } = renderWithProvider(<Send />, store);
       expect(queryByText('Next')).toBeNull();
     });
+
+    it('should render correctly even when a draftTransaction does not exist', () => {
+      const modifiedStore = {
+        ...baseStore,
+        send: {
+          ...baseStore.send,
+          currentTransactionUUID: null,
+        },
+      };
+      const store = configureMockStore(middleware)(modifiedStore);
+      const { getByPlaceholderText } = renderWithProvider(<Send />, store);
+      // Ensure that the send flow renders on the add recipient screen when
+      // there is no draft transaction.
+      expect(
+        getByPlaceholderText('Search, public address (0x), or ENS'),
```

Comment:
I know we use this in another place but selecting elements by placeholder always feels kinda weird.

Reply:
Probably just because I think of selecting elements by CSS paradigms, and you usually wouldn't select an element like this.  It's fine, just breaks my brain a little.